src/main/resources/api/service_openapi.yaml

```diff
@@ -702,7 +716,16 @@ components:
     ReferenceId:
       name: referenceId
       in: path
-      description: A UUID used to identify a data reference in a workspace
+      description: A UUID used to identify a data reference in a workspace.
+      required: true
+      schema:
+        type: string
+        format: uuid
+
+    ResourceId:
+      name: referenceId
```

Comment:
resourceId?

Reply:
Good catch. These are going to be merged in the new schema.

Reply:
@zloery I haven't been using the `@PathVariable` annotation in my controller. Should I? It's there for me in the generated parent class.

Reply:
Given that the annotation is in the generated api class, I don't think we need to explicitly add it to the controller.
I'm pretty sure we manually copied it in as we were implementing controllers for the first generated api class and it's been copied + pasted since.